app/views/ex_links/index.html.haml

```diff
@@ -0,0 +1,37 @@
+.col-md-12.margin-bottom-20
+  %h1 Listing External Links
+  .panel.panel-orange
+    %table.table.table-hover
+      %thead
+        %tr
+          %th Image
+          %th Label
+          %th Url
+          %th Tags
+          %th Test availability
+          %th Active
+          %th Expiration
+          %th Note
+          %th
+          %th
+          %th
+
+      %tbody
+        - @ex_links.each do |ex_link|
+          %tr
+            %td= image_tag ex_link.image.url(:thumb), width: 30
+            %td= ex_link.label
+            %td
+              %a{ :href => ex_link.url, :title => ex_link.label } #{ex_link.url}
```

Comment:
`%... \#{expression}` can be written without interpolation as `%...= expression`